cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -351,6 +353,28 @@ spec:
           securityContext:
             runAsUser: 0
 {{ end }}
+        {{- if .Values.kubecostModel.caCertsSecret }}
+        - name: "sslsetup"
+          image: {{ include "cost-model.image" . | trim | quote}}
+          imagePullPolicy: Always
+          securityContext:
```

Comment:
can we template this and add these as the defaults in values.yaml?

Reply:
thinking `kubecostModel.update-ca-trust.securityContext`

Reply:
agree here